src/fsharp/FSharp.Core.nuget/FSharp.Core.4.2.xxx.nuspec

```diff
@@ -42,15 +58,17 @@
         </dependencies>
     </metadata>
     <files>
-        <file src="FSharp.Core.dll"     target="lib/netstandard1.6" />
-        <file src="FSharp.Core.sigdata" target="lib/netstandard1.6" />
-        <file src="FSharp.Core.optdata" target="lib/netstandard1.6" />
-        <file src="FSharp.Core.xml"     target="lib/netstandard1.6" />
-        <file src="FSharp.Core.runtimeconfig.json" target="lib/netstandard1.6" />
-        <file src="FSharp.Core.sigdata" target="content/any/netstandard1.6" />
-        <file src="FSharp.Core.optdata" target="content/any/netstandard1.6" />
-        <file src="FSharp.Core.sigdata" target="runtimes/any/native/" />
-        <file src="FSharp.Core.optdata" target="runtimes/any/native/" />
-        <file src="FSharp.Core.xml"     target="runtimes/any/native/" />
+        <file src="coreclr/bin/FSharp.Core.dll"          target="lib/netstandard1.6/" />
+        <file src="coreclr/bin/FSharp.Core.sigdata"      target="lib/netstandard1.6/" />
+        <file src="coreclr/bin/FSharp.Core.optdata"      target="lib/netstandard1.6/" />
+        <file src="coreclr/bin/FSharp.Core.xml"          target="lib/netstandard1.6/" />
+        <file src="coreclr/bin/FSharp.Core.sigdata"      target="runtimes/any/native/" />
+        <file src="coreclr/bin/FSharp.Core.optdata"      target="runtimes/any/native/" />
```

Comment:
I think we can remove these files from ``runtimes/any/native`` for the 4.2.xx series. They are to support scenarios where the FSHarp.Core deployed is used to support further compilation at runtime.  Going forward these scenarios can expect an F# compiler that already knows about the integrated  sigdta/optdata.

However we should still keep the files in the ``lib`` directory to continue to support compilation at reference-time using this package with older F# compilers